system/src/Grav/Common/GPM/GPM.php

```diff
@@ -182,6 +208,28 @@ public function getInstalledThemes()
         return $this->installed['themes'];
     }
 
+    /**
+     * Checks if a Theme is enabled
+     *
+     * @param  string $slug The slug of the Theme
+     * @return bool
+     *   True if the Theme has been set to the default theme. False if installed, but not enabled. Null otherwise.
+     */
+    public function isThemeEnabled($slug)
+    {
+        $grav = new Grav();
+        $result = null;
+        $current_theme = $grav::instance()['config']['system']['pages']['theme'];
+        $theme_installed = $this->isThemeInstalled($slug);
+
+        if ($current_theme == $slug) {
```

Comment:
I'd do the same as with plugins. I would also move check for installed outside of this (and plugin) method and check it elsewhere..